terraform/aws/analytical-platform-development/cluster/iam-policies.tf

```diff
@@ -472,12 +472,13 @@ data "aws_iam_policy_document" "control_panel_api" {
     sid    = "CanManageSqsMessages"
     effect = "Allow"
     actions = [
+      "sqs:GetQueueAttributes",
       "sqs:GetQueueUrl",
       "sqs:DeleteMessage",
       "sqs:ReceiveMessage",
       "sqs:SendMessage"
     ]
-    resources = ["arn:aws:sqs::${var.account_ids["analytical-platform-development"]}:*"]
+    resources = ["arn:aws:sqs:eu-west-2:${var.account_ids["analytical-platform-development"]}:*"]
```

Comment:
If this is running in eu-west-2, I wouldn't hardcode the region but use a datasource to grab it. 
Example here:
https://stackoverflow.com/questions/51619602/terraform-lookup-aws-region#:~:text=I%20was%20looking%20for%20the%20same%20answer%20for%20a%20different%20problem.%20I%20wanted%20to%20get%20the%20region%20for%20a%20name%20of%20a%20role%2C%20I%20was%20able%20to%20get%20the%20info%20by%20doing%20this%3A